app/controllers/tasuku/taskables/questions/answers_controller.rb

```diff
@@ -6,6 +6,30 @@ class Taskables::Questions::AnswersController < ApplicationController
 
     before_action :set_question
 
+    def update
+      answer = Taskables::Question::Answer.find(params[:id])
+
+      return redirect_to :back, alert: I18n.t('tasuku.taskables.questions.answers.no_answers') unless params[:taskables_question_answer]
+
+      # Delete the old votes
```

Comment:
I find this comment is unnecessary. I think a good rule of thumb for commenting is when it's not easy to understand what the code does from reading it (rarely the case with Ruby) or to add context when it's not obvious _why_ you're doing something (like a weird hack for an edge case).